detection_rules/ecs.py

```diff
@@ -148,6 +150,28 @@ def get_non_ecs_schema():
     return load_etc_dump('non-ecs-schema.json')
 
 
+@cached
+def get_custom_schemas(stack_version: str):
+    """Load custom schemas if present."""
+    custom_schema_dump = {}
+    stack_schema_map = RULES_CONFIG.stack_schema_map[stack_version]
+
+    for schema, value in stack_schema_map.items():
+        if schema not in ["beats", "ecs", "endgame"]:
+            schema_path = RULES_CONFIG.stack_schema_map_file.parent / value
+            if schema_path.is_file():
+                custom_schema_dump.update(eql.utils.load_dump(str(schema_path)))
+            elif schema_path.is_dir():
+                custom_schema_dump.update(load_schemas_from_dir(schema_path))
```

Comment:
Should we enforce _only_ schema files vs a dir of schemas (requiring explicit paths for all). Unless you have an example where this would be used

Reply:
Personally, I like the idea of being able to specify a directory of schemas as I think it could be more convenient for CI.  

For example, if a customer has schema files that have dynamic names, e.g. my_schmea_v1.1, that could change to my_schema_1.2, they would not have to update their config file each time the name changed. Instead they could leave the config file the same and just specify a directory where my_schema_* would be located instead.

Reply:
but with the way it is written, won't it merge all the schemas together?

Reply:
Yes it would have that behavior, of merging them all into one. Maybe I am misunderstanding. Originally, I thought the question you were proposing was comparing someone being able to write:
```
8.14.0:
  beats: 8.12.2
  ecs: 8.11.0
  endgame: 8.4.0
  custom_one: schemas/custom/forte-schema_1.json
  custom_two: schemas/custom/forte-schema_2.json
  ...
```
vs
```
8.14.0:
  beats: 8.12.2
  ecs: 8.11.0
  endgame: 8.4.0
  custom: schemas/custom
```

In which case I would think we would want to support either?

Reply:
For flexibility, I like the ability to specify independent schemas or a folder of schemas. Definitely helps limit modifications to the config.

Reply:
Upon further discussion, removing the support for directories for the time being as it will likely create confusion for users. We currently do not support specifying more than one index for a file or directory. As such, if one specified a directory it would all be loaded under that index. While convenient for some, this abstraction of the index may create unforeseen and difficult to debug problems for many users. 

As such, I've removed the directory functionality from here.